### ISSUE TITLE
Change in how observations/models are named for web processing

### ISSUE THREAD

**Issue by hansbrenna** [metno/pyaerocom#130]
In CAMS61 there's a similarity between the variable specific obs data issue and the diurnal processing issue: The way observation data sets are named makes some things complicated. The name in the web interface now is derived directly from the key in the obs_config/model_config dictionaries. I see the potential for simplification if it was possible to optionally specify the web interface name (called `web_interface_name` below) inside each obs_config sub dict. This parameter would then be used when saving json files instead of the key in the dictionary. I have only really though about this for the observations, but it could be added for the models as well.

With support for dict specifiec obs_id, the setup for variable specific obs looks something like this:
```
'EEA-JP-1-5'        :   dict(obs_id={'concpm10' : 'GHOST.EEA.daily',
                                'concpm25' : 'GHOST.EEA.daily',
                                'concno2' : 'GHOST.EEA.daily',
                                'conco3' : 'GHOST.EEA.hourly'},
                             obs_vars=['concpm10','concpm25','concno2','conco3'],
                             obs_vert_type='Surface',
                             ignore_station_names={'concno2':no2_ignore_station_names,
                                                   'conco3':o3_ignore_station_names,
                                                   'concpm10':pm10_ignore_station_names,
                                                   'concpm25':pm25_ignore_station_names},
                             obs_filters={'set_flags_nan':True,
                                              'latitude':[30,70],'longitude':[-25,45],
                                              'Joly_Peuch_classification_code':[1,5],
                                              'ESDAC_Meybeck_landform_classification':landforms_to_include,}),
```
With this support I could turn this into (which is cleaner maybe?):
```
'EEA-JP-1-5'        :   dict(obs_id='GHOST.EEA.daily',
                             web_interface_name = 'EEA-JP-1-5'
                             obs_vars=['concpm10','concpm25','concno2'],
                             obs_vert_type='Surface',
                             ignore_station_names={'concno2':no2_ignore_station_names,
                                                   'conco3':o3_ignore_station_names,
                                                   'concpm10':pm10_ignore_station_names,
                                                   'concpm25':pm25_ignore_station_names},
                             obs_filters={filters}),

'EEA-JP-1-5_o3'        :   dict(obs_id='GHOST.EEA.hourly',
                             web_interface_name = 'EEA-JP-1-5'
                             obs_vars=['conco3'],
                             obs_vert_type='Surface',
                             ignore_station_names={'concno2':no2_ignore_station_names,
                                                   'conco3':o3_ignore_station_names,
                                                   'concpm10':pm10_ignore_station_names,
                                                   'concpm25':pm25_ignore_station_names},
                             obs_filters={filters}),
```
This would also solve my current issue where I'm thinking about how to finish the diurnal processing routine. Right now I'm doing like this:
```
obs_config = {

'GHOST'           :   dict(obs_id='GHOST.EEA.hourly',
                              obs_vars=['conco3'],
                              obs_vert_type='Surface',
                              obs_filters={'set_flags_nan':True,
                                              'latitude':[30,70],'longitude':[-25,45],}),

'GHOST-diurnal'           :   dict(obs_id='GHOST.EEA.hourly',
                             obs_vars=['conco3'],
                             obs_vert_type='Surface',
                             ts_type='hourly',
                             diurnal_only = True,
                             obs_filters={'set_flags_nan':True,
                                              'latitude':[30,70],'longitude':[-25,45],}),
},
```
Which doe not really work when the resulting json files are processed for the web site. If we had the possibility of setting the web interface name I could do just set that to `web_interface_name = 'GHOST'` in the specification of GHOST-diurnal.

**Comment by jgliss** [comment]
@hansbrenna that sounds like a useful extension! If I understand correctly, in that case, `obs_id` would always be a string, since we could handle variable specific obs IDs via different entries in the `obs_config`. Right?

**Comment by hansbrenna** [comment]
Yes, obs_id would always be a string and complicated processing would be done by splitting the config into multiple pieces.

**Comment by jgliss** [comment]
I think that sounds like a cleaner way to handle the processing. I did not like the idea of having obs_id anything other than a string. @hansbrenna will you be working on this? In case yes, please also keep in mind #132 and #133. If you want, we can also have a short meeting, where I can explain a little more about the test-suite and conventions that we use. @ejgal may also be of help!

**Comment by hansbrenna** [comment]
I can work on it as I'm the one who needs it most urgently. 

I don't know anything about the kind of testing we're using so I would need a meeting or something to get started.

**Comment by hansbrenna** [comment]
I'm almost done with this. Testing it for the first use case described above now. If it works I'll make a pull request to my main development branch when the testing is done. 

One issue which requires some thinking now is that this change in the processing breaks the table under the informations pane in the web interface due to the fact that obs_ids and web_interface_names/obs_config-ids are no longer linked unambiguously.

**Comment by jgliss** [comment]
@hansbrenna I marked this as almost Done, as I believe this has been implemented in PR #214, correct?
